test_exchange/README.md

```diff
@@ -0,0 +1,30 @@
+# Test Exchange
+
+Service which can emulate an exchange, and is to be used with the eur2ccd services test exchange parameter.
+It maintains a queue of values, which is serves through the `/rate` endpoint. If the queue is empty, the value specified by the resort_value, is returned instead.
+
+Has 2 parameters:
+
+ * `port` (environment variable: `TEST_EXCHANGE_PORT`): Port at which the exchange is served.
+ * `resort-value` (environment variable: `TEST_EXCHANGE_RESORT_VALUE`): The exchange rate, which is returned, when the queue is empty.
+
+Has 3 endpoints:
+
+ * `GET /rate`: get an exchange rate (this should pointed to by the eur2ccd service.
+ * `POST /add`: Expects a body that is a json array of floats, which will be added to the queue of values, which is served on `/rate`.
```

Comment:
Would it make sense here to allow any JSON array (not just of floats, but of any Values), to test that if the API spec is violated, the service can handle it.

Reply:
Sure, I have changed it to allow this.